dotnet3.5/dynamodb/README.md

```diff
@@ -0,0 +1,464 @@
+<!-- Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+     SPDX - License - Identifier: Apache - 2.0 -->
+# Amazon DynamoDB code examples in C\#
+
+This folder contains code examples for moving from SQL to the Amazon DynamoDB service,
+as described in the Amazon DynamoDB Developer Guide at
+[From SQL to NoSQL](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/SQLtoNoSQL.html).
+
+All of these code examples are written in C#, 
+using the V3.5 version of the AWS SDK for .NET.
+Getting the 3.5 version of the SDK is straightforward using the command line 
+from the same folder as your ```.csproj``` file.
+For example, to add a reference to the latest (V3.5) version of Amazon DynamoDB
```

Comment:
This is confusing because it sounds like V3.5 is referring to the version of DynamoDB. I'd just remove the (V3.5).

Reply:
Fixed